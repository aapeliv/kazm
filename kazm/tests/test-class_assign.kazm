class Person {
	int x;
	double y;
};

int main(){
	Person a;
	Person b;
	a.x = 40;
	a.y = 30.0;
	b = a;
	int_println(b.x);
	double_print(b.y);
	println("");

	b.x = b.x + b.x;
	a = b;
	int_println(a.x);
}
