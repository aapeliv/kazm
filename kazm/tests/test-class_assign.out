40
30.000000
80
